src/transformStateWithClones.js

```diff
@@ -7,7 +7,44 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyOfState = { ...state };
+  const anwser = [];
+
+  for (const actionObjects of actions) {
+    for (const action in actionObjects) {
+      if (actionObjects[action] === 'addProperties') {
+        for (const data in actionObjects['extraData']) {
```

Comment:
```suggestion
        for (const data in actionObjects.extraData) {
```
Ти точно знаєш назву ключа. Для чого потрібні квадраті дужки?